include/frontier/overlay_80_022340E8.h

```diff
@@ -64,7 +64,7 @@ u32 BattleArcadeData_Alloc(SaveData *saveData, u32 a1, u8 a2, u32 a3, u32 a4, u3
 void BattleArcadeData_Init(void *a0, u32 a1);
 void BattleArcadeData_Free(void *a0);
 u8 BattleArcade_GetWonBattlePoints(ArcadeContext *ctx, Party *playerParty, Party *opponentParty, int a3);
-void ov80_02235364(Party *a0, Party *a1, int a2, int a3);
+void ov80_02235364(Party *party, Party *playerParty, int a2, int a3);
```

Comment:
what's the difference between these two `Party *` parameters?

Reply:
not 100% sure since that specific function isn't decompiled yet, it's something related to the roulette event that swaps (some) of your party mons with the opponents party mons though